mpa/seg/inferrer.py

```diff
@@ -108,16 +109,26 @@ def infer(self, cfg):
             _ = load_checkpoint(model, cfg.load_from, map_location='cpu')
 
         # Inference
-        model = MMDataParallel(model, device_ids=[0])
+        model.eval()
+        eval_model = MMDataParallel(model, device_ids=[0])
```

Comment:
Is CPU inference considered? I think device_ids argument can cause some error with cpu-only machine.